doc/source/reference/c-api.array.rst

```diff
@@ -1904,7 +1904,7 @@ Item selection and manipulation
         all values are clipped to the region [0, len(*op*) ).
 
 
-.. c:function:: PyObject* PyArray_Sort(PyArrayObject* self, int axis)
+.. c:function:: PyObject* PyArray_Sort(PyArrayObject* self, int axis, NPY_SORTKIND)
```

Comment:
While this is legal C, it would be good to add the argument name `kind` here too.

Reply:
```suggestion
.. c:function:: PyObject* PyArray_Sort(PyArrayObject* self, int axis, NPY_SORTKIND* sort)
```

Reply:
I don't think your `*` there is correct - I doubt we pass a pointer to an enum.